Skills.md

```diff
@@ -38,6 +39,24 @@ JSON
   ]
 }
 ```
+If the return_relevant_skills request parameter is set to true, the reponse will look like this:
+
+```
+{
+	"response": [{

```

Comment:
we should fix up the formatting for readability, and I would do something like this to indicate that the relevant_skills list is longer than shown:
```
"relevant_skills": [
  {
  }, (...)
]